spec/lib/project_copier_spec.rb

```diff
@@ -41,6 +41,23 @@
       expect(copied_project.configuration['source_project_id']).to be(project.id)
     end
 
+    it 'sets the value of each excluded attribute to their default' do
+      # N.B, to be updated once ProjectCopier::EXCLUDE_ATTRIBUTES values are updated aswell
+      default_values = [0, 0, nil, 0.0, 0, 0]
+      # excluding launched_row_order and beta_row_order as they are primary keys and don't have a fixed default value
```

Comment:
Unrelated, but I dont think `launched_row_order` and `beta_row_order` are primary keys. 

I think for projects the only primary key is `id`. BUT I do think `launched_row_order` and `beta_row_order` are indexes of project. (See: [here](https://github.com/zooniverse/panoptes/blob/b91a4c27344c4cf89592e52c1d3977e3c29d966e/db/structure.sql#L3205) and  [here](https://github.com/zooniverse/panoptes/blob/b91a4c27344c4cf89592e52c1d3977e3c29d966e/db/structure.sql#L3233) )